k8s/prod/common/ia/home-office-integration-api.yaml

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
new applications are required to use kustomize base design: https://github.com/hmcts/cnp-flux-config/blob/master/docs/app-deployment.md

are you able to do this before go-live?

Reply:
It will be a bit stretch task because the release is tomorrow. We have another release next week. Can I apply kustomize design together with next service (ia-case-payments-api) release?

Reply:
yes sure